src/selfplay/game.cc

```diff
@@ -217,6 +205,7 @@ void SelfPlayGame::Play(int white_threads, int black_threads, bool training,
         }
         if (edge.GetMove(tree_[idx]->IsBlackToMove()) == move) {
           cur_n = edge.GetN();
+          played_wl = edge.GetWL(best_eval.wl);
```

Comment:
The fall through value here should be fallback to parent wl rather than best?

Reply:
I think fallback to `best_wl` is fine in the application we are using it for, as it would assume that we picked the no temp move, and therefore never trigger a blunder detection.

Reply:
if the fall back is triggered, that means we've picked a move with 0 visits... 
Which should never happen (unless using a *positive* temp offset), so I'd rather go with being more consistent.

Reply:
Ok, I see & agree with that reasoning.